server/internal/usecase/repo/project.go

```diff
@@ -18,6 +18,7 @@ type ProjectFilter struct {
 	Offset      *int64
 	SearchField *string
 	Visibility  *string
+	Topics      *[]string
```

Comment:
I think arrays shouldn’t allow nil; an empty array should be sufficient.

Reply:
done